src/core/src/pass/serialize.cpp

```diff
@@ -916,10 +916,21 @@ void serialize_rt_info(pugi::xml_node& root, const std::string& name, const ov::
         child.append_attribute("name").set_value(name.c_str());
     }
     if (data.is<std::shared_ptr<ov::Meta>>()) {
```

Comment:
```suggestion
    if (auto meta = data.as<std::shared_ptr<ov::Meta>>()) {
```

Reply:
We got an exception if it is not a ov::Meta. I would like keep this part without exception handling.